ucl-university-college-apa.csl

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded" default-locale="da-DK">
-  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
   <info>
     <title>UCL University College - APA</title>
-    <title-short>UCL - APA</title-short>
+    <title-short>APA - UCL</title-short>
     <id>http://www.zotero.org/styles/ucl-university-college-apa</id>
-    <link href="http://www.zotero.org/styles/ucl-university-college-apa" rel="self"/>
```

Comment:
This also needs to be http://www.zotero.org/styles/ucl-university-college-apa

Reply:
Thank you! I think I got it now :)